### ISSUE TITLE
Help: Synology Docker container qmcgaw-ddns stopped unexpectedly.

### ISSUE THREAD

**Issue by kruu51** [qdm12/ddns-updater#93]
**TLDR**: *Describe your issue in a one liner here*

1. Is this urgent?

    - [ ] Yes
    - [x] No

2. What DNS service provider(s) are you using?

    - [ ] Cloudflare
    - [ ] DDNSS.de
    - [ ] DonDominio
    - [ ] DNSPod
    - [ ] Dreamhost
    - [ ] DuckDNS
    - [ ] DynDNS
    - [x] GoDaddy
    - [ ] Google
    - [ ] Infomaniak
    - [ ] Namecheap
    - [ ] NoIP

3. What's the version of the program?

  latest

4. What are you using to run the container?

    - [x] Docker run
    - [ ] Docker Compose
    - [ ] Kubernetes
    - [ ] Docker stack
    - [ ] Docker swarm
    - [ ] Podman
    - [ ] Other:

5. Extra information

Logs:

```log

```

Configuration file:

https://www.dropbox.com/s/rliolqnveg6ipgh/qmcgaw-ddns.html?dl=0

Host OS:
Synology DSM

**Comment by qdm12** [comment]
Did you change the json configuration file? That seems like an error parsing the JSON file. Can you share the content of the file (and remove credentials)?

**Comment by kruu51** [comment]
I connect with SSH into my Synology NAS, but I dont know where the config file of the container is.

**Comment by qdm12** [comment]
You need to create a config.json file as described in the readme and then bind mount in the container. Please have a look at the readme. I understand it's not the easiest way to get started but haven't got the time to  make an interactive UI to set things up!
